lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
The name `records` seems to represent `returns the event entries`.
`num_records` or `size`, rubish, is better for me.